usecases/backup/backend.go

```diff
@@ -65,52 +65,55 @@ const (
 
 var _NUMCPU = runtime.NumCPU()
 
-type objStore struct {
-	b        modulecapabilities.BackupBackend
-	BasePath string
+type ObjectStore struct {
+	Backend modulecapabilities.BackupBackend
```

Comment:
can we keep this variable private? I think that we don't need to expose it anywhere

Reply:
libweaviate

Reply:
Can we then make those structs private again and create a method:
```
func NewNodeStore(backend modulecapabilities.BackupBackend, backupId, bucket, path string) nodeStore {
	return nodeStore{objectStore: objectStore{backend, backupId, bucket, path}}
}
```
then in libwaeaviate instead of doing this:
```
ns := backup.NodeStore{
				backup.ObjectStore{
					backend,
					"test",
					"", "",
				},
			}
```
you could do simply this:
```
ns := backup.NewNodeStore(backend, "test", "", "")
```
and we wouldn't have to make those structs as public